src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -235,7 +241,12 @@ def add_realpath(path):
         realpaths.add(realpath)
 
     cp = configparser.SafeConfigParser()
-    cp.readfp(StringIO(self.DEFAULT_COVERAGE_CONFIG))
+    coverage_config_file = self.get_options().coverage_config_file
+    if coverage_config_file is not None:
+      with io.open(coverage_config_file, 'rt') as fp:
+        cp.readfp(fp)
```

Comment:
This deserves a test. Coverage tests already exist [here](https://github.com/jsirois/pants/blob/master/tests/python/pants_test/backend/python/tasks2/test_pytest_run.py#L494) for inspiration / test infra to proceed with.